packages/client/src/components/LoginPanel.tsx

```
@@ -74,86 +163,144 @@ export const LoginPanel: React.FC = () => {
+          fontWeight: 700,
+          fontFamily: fonts.body,
+          padding: '11px 16px',
+          borderRadius: radii.md,
+          border: 'none',
+          boxShadow: shadows.low,
+          cursor: isLoading ? 'not-allowed' : 'pointer',
+          opacity: isLoading ? 0.55 : 1,
+          marginBottom: 14,
+          transition: 'opacity 0.15s',
+        }}
       >
```

Biome (2.5.0)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)